pythonbuild/cpython.py

```diff
@@ -511,7 +511,10 @@ def derive_setup_local(
             )
 
             if target_match and (python_min_match and python_max_match):
```

Comment:
The changes in this file in support of an ergonomic expression in https://github.com/astral-sh/python-build-standalone/pull/443/commits/d0521dd254920296e788feae6441cc89a5c724d8